pkg/server/auth/jwt.go

```diff
@@ -139,6 +119,7 @@ func (pg *JWTAdminCookiePrincipalGetter) Principal(r *http.Request) (*UserPrinci
 		return nil, nil
 	}
 
+	// TODO: Fix This
 	return parseJWTAdminToken(pg.verifier, cookie.Value)
```

Comment:
So this one seems to get in the way. If you don't specify

```yaml
additionalArgs:
  - --auth-methods=oidc
```

In the values then we always end up here, even when logging in w/ OIDC, and so we end up w/ the `aud` claim. Which is a bit funny.

Reply:
Sorry my bad `sub`, not `aud`.

Reply:
This was `devMode` short-circuiting things